crates/task-impls/src/quorum_vote/mod.rs

```diff
@@ -307,6 +357,10 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES> + 'static, V: Versions> Handl
                         return;
                     }
                     leaf = Some(proposed_leaf);
+                    // Proposed and parent block are the same. We don't care about VID and DAC.
+                    if is_same_block_proposed {
+                        break;
+                    }
```

Comment:
is there any danger we get e.g. a `DaCertificateValidated` causing us to cancel the quorum vote task? (it looks like no just from quickly looking at the code but wanted to check)

Reply:
I think I just meant that in the way the `quorum_vote` task is currently structured, I think receiving a `DaCertificateValidated` in line 369 can in principle cause the full task to abort (if I'm reading it correctly?). but I'm not 100% clear on what the code paths here are

but what you said makes sense and answers the question I think: we shouldn't receive a DAC/VID because we would spawn this task just off the justify qc and immediately exit if it's an epoch transition